### ISSUE TITLE
🐞 - tui-select wrong screen reader output

### ISSUE THREAD

**Issue by juanfran** [taiga-family/taiga-ui#3784]
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://taiga-ui.dev/stackblitz

### Is this issue blocking you?

Non-Blocking

### Description

When NVDA is used as a screen reader, the `tui-select` component is read as read-only, even when it's not. It also doesn't indicate that it's a combo box in its collapsed state. 

![a11y-tui](https://user-images.githubusercontent.com/905195/222369653-65e971de-a675-4dc6-9501-cef342e390d4.gif)

Expected output should be 'Choose your hero combo box Luke Skywalker collapsed'

### Angular version

15.0.0

### Taiga UI version

3.18.0

### Which browsers have you used?

- [X] Chrome
- [X] Firefox
- [ ] Safari
- [X] Edge

### Which operating systems have you used?

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

**Comment by juanfran** [comment]
Sure! I believe that the web examples should be checked with a screen reader. I remember encountering some of them where it read [object Object], but I'm unsure of how to fix them without touching `tui`. Additionally, the tui-data-list includes an HTML <div tabindex="0" class="t-trap"></div> which may cause confusion when navigating with a keyboard+screen reader. Probably the icons within the buttons should have an aria-hidden attribute. We are working to make an app that is compatible with NVDA, so we can let you any improvements we can think of.

**Comment by waterplea** [comment]
That would be great. Please post theme in this issue: https://github.com/Tinkoff/taiga-ui/issues/4015

**Comment by waterplea** [comment]
This is fixed with new textfield based select, legacy will remain as it is now.
